sdks/python/apache_beam/utils/dependency.py

```diff
@@ -476,7 +476,12 @@ def _download_pypi_sdk_package(temp_dir):
   """Downloads SDK package from PyPI and returns path to local path."""
   # TODO(silviuc): Handle apache-beam versions when we have official releases.
   import pkg_resources as pkg
-  version = pkg.get_distribution(GOOGLE_PACKAGE_NAME).version
+  try:
+    version = pkg.get_distribution(GOOGLE_PACKAGE_NAME).version
+  except pkg.DistributionNotFound:
+    raise RuntimeError('Please set --sdk_location or install a valid '
```

Comment:
`--sdk_location command-line option` instead of just `--sdk_location`. Because this is how we refer to the same thing in a related error in this same file.